app/decorators/models/solidus_address_name/address_decorator.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module SolidusAddressName
+  module AddressDecorator
+    alias_attribute :first_name, :firstname
+    alias_attribute :last_name, :lastname
+    alias_attribute :full_name, :name
```

Comment:
Are you sure this is working?
AFAIK `alias_attributes`(and class_methods in general) are only available inside the `prepended` method.

E.G.

```
module SolidusAddressName
  module AddressDecorator
    def self.prepended(base)
      base.alias_attribute :first_name, :firstname
      ...
    end
    ....
  end
end
```